samples/emoji-search/ios/app/EmojiSearchApp/ChildrenBinding.swift

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright (C) 2022 Square, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import UIKit
+import shared
+
+class ChildrenBinding: Redwood_widgetWidgetChildren {
```

Comment:
PROBABLY need a Kotlin implementation in redwood-treehouse. Will need help from @kylebshr on the API.

Reply:
I can take this and just reuse the API for now. We already have the need in more than one sample.

Reply:
https://github.com/cashapp/redwood/pull/364